rel-eng/push-packages-to-obs.sh

```diff
@@ -259,6 +259,8 @@ while read PKG_NAME; do
   fi
   # to lowercase with ",," and replace spaces " " with "-"
   PRODUCT_VERSION=$(echo ${PRODUCT_VERSION,,} | sed -r 's/ /-/g')
+  # Remove leading zero from Uyuni release and add potentially missing micro part
+  SEMANTIC_VERSION=$(echo ${PRODUCT_VERSION} | sed 's/\([0-9]\+\)\.0\?\([1-9][0-9]*\)\(\.\([0-9]\+\)\)\?\( .\+\)\?/\1.\2.\4\5/' | sed 's/\.$/.0/')
```

Comment:
```
$ PRODUCT_VERSION='4.4 Beta1'
$ SEMANTIC_VERSION=$(echo ${PRODUCT_VERSION} | sed 's/\([0-9]\+\)\.0\?\([1-9][0-9]*\)\(\.\([0-9]\+\)\)\?\( .\+\)\?/\1.\2.\4\5/' | sed 's/\.$/.0/')
$ echo $SEMANTIC_VERSION 
4.4. Beta1
```
I think this still needs to handle the strings.

Check what's dome for the Docker images. IIRC it replaces ` `with `-` and changes everything to lowercase.

Works for Uyuni, however:
```
$ PRODUCT_VERSION='2022.07'
$ SEMANTIC_VERSION=$(echo ${PRODUCT_VERSION} | sed 's/\([0-9]\+\)\.0\?\([1-9][0-9]*\)\(\.\([0-9]\+\)\)\?\( .\+\)\?/\1.\2.\4\5/' | sed 's/\.$/.0/')
$ echo $SEMANTIC_VERSION 
2022.7.0

Reply:
https://github.com/uyuni-project/uyuni/commit/5f69a7f44824cad8acd48c1bb4fc4202c3316c6f shows we include the `.0` in major releases and you forgot to run `PRODUCT_VERSION=$(echo ${PRODUCT_VERSION,,} | sed -r 's/ /-/g')` in your test.

Reply:
Yes, you are right. Seems to be working fine now.